variables.pkr.hcl

```diff
@@ -77,7 +77,19 @@ variable "instance_type" {
 }
 
 variable "ami_name_prefix" {
-  description = "The prefix to use when creating the AMI name. i.e. - `<ami_name_prefix>-<eks_version>-<timestamp>"
+  description = "The prefix identifying the source AMI to use when creating the AMI. i.e. - `<ami_name_prefix>-<eks_version>-<timestamp>"
```

Comment:
I think it would be better to just update the variable description to make it clear that its the name prefix that is used to both lookup the base AMI as well as the name prefix used for the resulting AMI